multy_core/src/ethereum/ethereum_transaction_builder_multisig.cpp

```diff
@@ -237,10 +300,48 @@ class MultisigPaymentRequestActionTransactionBuilder : public MultisigTransactio
     {
         validate(MULTY_CODE_LOCATION);
 
-        // TODO: make a transaction
-        return nullptr;
+        TransactionPtr transaction;
+        ::make_transaction(&m_account, reset_sp(transaction));
+        {
+            Properties& source = transaction->add_source();
+            source.set_property_value("amount", BigInt(1000000 )* 1000000 * 1000000);
+        }
+
+        {
+            Properties& destination = transaction->add_destination();
+            destination.set_property_value("address", EthereumAddress::to_string(*m_wallet_address));
+            destination.set_property_value("amount", BigInt(0)); // How much transfer to SC
+        }
+
+        ::transaction_set_message(transaction.get(),make_message().get());
+        return TransactionPtr(transaction.release());
     }
 
+    BinaryDataPtr make_message() const override
+    {
+        static const std::unordered_map<size_t, EthereumContractMethodHash> METHODS =
+        {
+            {CONFIRM, {0xc0, 0x1a, 0x8c, 0x84}},
+            {REJECT,  {0x20, 0xea, 0x8d, 0x86}},
+            {SEND,    {0xee, 0x22, 0x61, 0x0b}}
+        };
+
+        const auto method = METHODS.find(*m_action);
+        if (method == METHODS.end())
+        {
+            THROW_EXCEPTION2(ERROR_INVALID_ARGUMENT,
+                    "Invalid method.")
+                    << " action: \"" << m_action.get_name() << "\".";
+        }
+
+        EthereumPayloadDataStream list;
+
+        list << method->second;
+        EthereumPayloadAsUint256Stream variable;
+        variable << *m_request_id;
+        list << variable;
+        return list.get_content();
```

Comment:
code style: please ensure that `return` always has one blank line above (here and every other place in PR).